hack/kubevirt-builder/ci-test.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
We might integrate this into the Makefile later on or move it into `test/` folder. There is another test script already.

Reply:
for now, I've moved all files necessary for CI to an isolated dir, `hack/ci`